app/components/Settings/EncryptQR/EncryptQR.jsx

```diff
@@ -0,0 +1,83 @@
+// @flow
+import React, { Component } from 'react'
+
+import Button from '../../Button'
+import { ROUTES } from '../../../core/constants'
+import styles from './EncryptQR.scss'
+import FullHeightPanel from '../../Panel/FullHeightPanel'
+import CloseButton from '../../CloseButton'
+import BackButton from '../../BackButton'
+import ConfirmIcon from '../../../assets/icons/confirm.svg'
+import CopyIcon from '../../../assets/icons/copy.svg'
+import CheckIcon from '../../../assets/icons/check.svg'
+import AddIcon from '../../../assets/icons/add.svg'
+import withCopyCanvasToClipboard from '../../../hocs/withCopyCanvasToClipboard'
+
+type Props = {
+  encryptedWIF: string,
+  handleCopy: (?HTMLCanvasElement, string, ?boolean) => Promise<void>,
+  handleCreateCanvas: (?HTMLCanvasElement, string) => any,
+  copied: boolean,
+}
+
+class EncryptQR extends Component<Props> {
+  encryptedCanvas: ?HTMLCanvasElement
+
+  componentDidMount() {
+    const { encryptedWIF } = this.props
+    this.props.handleCreateCanvas(this.encryptedCanvas, encryptedWIF)
+  }
+
+  render() {
+    return (
+      <FullHeightPanel
+        headerText="Encrypted QR Code"
+        renderInstructions={false}
+        headerContainerClassName={styles.headerIconMargin}
+        renderHeaderIcon={() => <CheckIcon />}
+        renderCloseButton={() => <CloseButton routeTo={ROUTES.SETTINGS} />}
```

Comment:
I think it would be a good idea to invoke `resetEncryptedWIF` on close here. Resetting the state after closing the panel seems to make the most sense from a UX perspective rather than having the user click "Encrypt another private key"